.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: 
+  push:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Python ${{ matrix.version }}
+    strategy: 
+      matrix:
+        version: [2.7, 3.4, 3.5, 3.6]
```

Comment:
Wondering if we should pop in more versions to test?

Reply:
We'll keep 2.7, and for the 3 series I think we should cover the latest release, and the oldest supported, so 3.6 and 3.9. I don't think we need to cover the in between ones.